Classes/FrontEnd/Editor.php

```diff
@@ -2,6 +2,8 @@
 
 declare(strict_types=1);
 
+use Sys25\RnBase\Configuration\Processor;
```

Comment:
Please let's use this class as `ConfigurationProcessor` so the name/alias makes the context more clear.